android/build.gradle

```diff
@@ -66,9 +86,14 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.annotation:annotation:1.3.0'
 
-    api("com.revenuecat.purchases:purchases:$purchases_version")
-    api("com.revenuecat.purchases:purchases-core-common:$purchases_version")
-    api("com.revenuecat.purchases:purchases-core-utils:$purchases_version")
+    latestDependenciesApi "com.revenuecat.purchases:purchases:$purchases_version"
+    latestDependenciesApi "com.revenuecat.purchases:purchases-core-common:$purchases_version"
+    latestDependenciesApi "com.revenuecat.purchases:purchases-core-utils:$purchases_version"
+    
+    // Uncomment once we upgrade to 5.0.0
```

Comment:
This won't work until we upgrade the dependency to 5.0.0, which is in progress in another PR.